var/spack/repos/builtin/packages/gapfiller/package.py

```diff
@@ -33,7 +33,7 @@ def url_for_version(self, version):
 
     def patch(self):
         with working_dir('.'):
-            files = glob.iglob("*.pl")
+            files = glob.iglob("**.pl")
```

Comment:
First of all:
```suggestion
            files = glob.iglob("**.pl", recursive=True)
```
Also, it looks like support for this was added in Python 3.5, so this won't work for Python 2. You'll have to use `os.walk` instead. If you look in `lib/spack/llnl/util/filesystem.py`, I think you could use the `find` function here. It works pretty similarly to the UNIX command-line `find`.

Reply:
I modified the recipe to stick to the original `glob.glob` by explicitly adding the bwa subdirectory; as there are very few perl files it needs to find I think this is the most succinct approach.